### PR TITLE
Readd self-examination and add wounds descriptions

### PR DESCRIPTION
**Pull request by corp-0**
### Purpose
Readd self-examination interaction. It will return the classic description you found when examininig others as seen on this recent screenshot from tg:
unknown

Adds wounds descriptions for each body part to the new expanded view of the examination window. The numbers that determine the message might need a little tweaking in the future.

Adds a short one word status description to be seen on the status part of the new examination window.

### Notes:
I added two new extensions. 
Capitalize() is a string extension that will, welp, capitalize the word so the first letter of the string is Upcase.
IsBetween() is a IComparable extension that will return true if the IComparable is between the given range. This works for all IComparables, like numbers (of any type) and dates.

### Changelog:
CL: Readds self-examination
CL: Adds new wounds description per body part visible on the new extended view of the examination window.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs [hunk 1]
```diff
@@ -1048,13 +1054,49 @@ public string GetExamineText()
...
 		}
 
 		return healthString;
 	}
 
+	public string GetShortStatus()
+	{
+		if (IsDead)
+		{
+			return "DEAD";
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs:1077]: could add extra if for is SSD and dead?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs:1077]: Yes I can. I didn't know if we wanted to add information that specific, is it in character information to know how someone is dead and has no soul?

Comment (corp-0) [reply UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs:1077]: done

**Comment by corp-0** [comment]
wait until #5879 is merged because it this will conflict then. I don't mind solving the conflict.

**Comment by corp-0** [comment]
I'm gonna do a gamer move and speed merge this so we can trigger a build sooner

**Pull request merged**